infection.json5

```diff
@@ -10,14 +10,17 @@
             "FileSystem/DummySymfony5FileSystem.php",
             "FileSystem/DummySymfony6FileSystem.php",
             "CustomMutator/templates/__Name__.php",
-            "CustomMutator/templates/__Name__Test.php",
+            "CustomMutator/templates/__Name__Test.php"
         ]
     },
     "logs": {
         "stryker": {
             "report": "master"
         }
     },
+    phpStan: {
+        configDir: "devTools"
+    },
```

Comment:
@staabm this is the fix for https://github.com/infection/infection/pull/2165#issuecomment-2960429463

Reply:
confirmed, works now for me - thanks